src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,58 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "cinema_halls")
+public class CinemaHall {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private Integer capacity;
+    private String description;
+
+    public CinemaHall() {
```

Comment:
Don't create a default constructor when it is not needed. If Hibernate requires a no-argument constructor, it can be package-private to prevent its use outside of the package.

Reply:
+